docs/getting-started/operating-systems.md

```diff
@@ -0,0 +1,59 @@
+# Operating System Compatibility and Differences 
+
+This page was last updated for v0.8.2
+
+---
+
+The purpose of this page is describe which binaries are available for what operating systems. It will also cover the key differences for running the cross-platform command-line applications.
+
+---
+
+## The Applications
+
+**Paymetheus**: The only *complete* GUI client as of v0.8.2. <br />
+**dcrd**: The node daemon, this command-line application handles block management and consensus. <br />
+**dcrwallet**: The wallet daemon, this command-line application handles address and transaction management. <br />
+**dcrctl**: The remote procedure call (RPC) client, this command-line application is used to control dcrd and dcrwallet via RPC commands. <br />
+**Decrediton**: BETA GUI client as of v0.8.2.
+
+|           | Paymetheus | dcrd | dcrwallet | dcrctl | Decrediton |
+| ---------:|:----------:|:----:|:---------:|:------:|:-----------:|
+| Windows   | X          | X    | X         | X      |             |
+| macOS     |            | X    | X         | X      | X           |
+| Linux     |            | X    | X         | X      | X           |
```

Comment:
If you are being really complete, you could add Other UNIX for dcrd, dcrwallet, and dcrctl since we support various BSDs and Solaris for them but not for the GUIs.